### DISCUSSION TITLE
Is Tasmota the solution for my Timed LED Lamps projects?

### DISCUSSION THREAD

**Discussion by leotordo** [arendst/Tasmota#22706] [Q&A]
Hello everyone,

I'm asking for some advice because I don't know if Tasmota is the right solution for my project.

I need to build a certain number of LED lamps that turn on when the power is connected and turn off after a programmed time. The time will be set via an Android/iOS app. I'll be using an ESP32 module, and an output pin will control the enable of the LEDs.

Could Tasmota be the solution? Can I do this without a central hub or do I still need a PC/Raspberry Pi? Do the lamps need to be configured one by one, or can they auto-configure the network?

Please forgive my ignorance on the matter.

Thank you,

Stefano

**Comment by Jason2866** [comment]
Tasmota has no native APP support. APP can be realized via a Home Automation system.
There is no easy short answer for your question. It is possible to achieve what you want to do.
BUT you have to invest time and learn how all this works. There is no quick easy solution for this with Tasmota.

**Reply by sfromis** [reply]
While no app, it is *possible* to use a separate http (like a browser) session to each. Not a great way with multiple lamps. If they are to always act in sync, Tasmota has a device groups feature using UDP to replicate actions across a group. No auto-configuration (from zero) "out of the box", unless/until you do something to facilitate such.
